.github/workflows/sdl_core_github_ci.yml

```diff
@@ -0,0 +1,41 @@
+name: SDL Core Build
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Init Submodules
+      run: git -C ${{ github.workspace }} submodule update --init --recursive
+    # Install Dependencies
+    - name: Apt Get Dependencies
+      run: sudo apt-get update && sudo apt-get install libssl-dev libbluetooth3 libbluetooth-dev libudev-dev cmake html2text lcov git cmake automake1.11 build-essential libavahi-client-dev sqlite3 libsqlite3-dev libgtest-dev bluez-tools libpulse-dev libusb-1.0.0-dev cppcheck python3-pip python3-setuptools && sudo apt-get install -f clang-format-6.0
+    - name: Setup CMAKE
+      uses: jwlawson/actions-setup-cmake@v1.8
+      with:
+        cmake-version: '3.16.3'
+    # Check Style
+    - name: Check Style
+      run: ${{ github.workspace }}/tools/infrastructure/check_style.sh
+    # Build Project
+    - name: Make Directories
+      run: cd ${{ github.workspace }} && cd ../ && mkdir core_3rd_party && mkdir build && cd build
+    - name: Set 3rd Party Path
+      run: echo "THIRD_PARTY_INSTALL_PREFIX=$(echo $(cd ../core_3rd_party && pwd))" >> $GITHUB_ENV
+    - name: Set 3rd Party ARCH Path
+      run: echo "THIRD_PARTY_INSTALL_PREFIX_ARCH=$(echo $(cd ../core_3rd_party && pwd))" >> $GITHUB_ENV
+    - name: Cache 3rd Party
+      id: core-3rd-party
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/../core_3rd_party
```

Comment:
Seems the cache has an issue with this:
```
Post job cleanup.
Warning: Invalid pattern '/home/runner/work/sdl_core/sdl_core/../core_3rd_party'. Relative pathing '.' and '..' is not allowed.
```
Maybe just put the core_3rd_party directory in the source directory?